department-of-veterans-affairs/va.gov-cms
# VACMS-18805: Configurations for updating KB search.

## Description
This work updates the configurations for the knowledge base search functionality.

Closes #_18805_.

## Testing done
Local search testing with A/B results for different scenarios outlined in ticket https://github.com/department-of-veterans-affairs/va.gov-cms/issues/8136

## QA steps

What needs to be checked to prove this works?
Search within the knowledge base for something you need.
What needs to be checked to prove it didn't break any related things?
Perform basic searches in knowledge base to make sure it is behaving as expected.
What variations of circumstances (users, actions, values) need to be checked?
Add in some of the stopwords located [here](https://raw.githubusercontent.com/stopwords-iso/stopwords-en/master/stopwords-en.txt) to see if they populate the search results (they shouldn't).

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
